Core/Lexer/Tokenization/TokenKind.cs

```diff
@@ -33,18 +33,6 @@ public enum TokenKind : ushort
         [Keyword("readonly")]
         ReadOnly,
 
-        /// <summary>
-        ///     The 'deprecated' keyword which is reserved by the compiler
-        /// </summary>
-        [Keyword("deprecated")]
-        Deprecated,
```

Comment:
Why are these tokens being removed?

Reply:
I changed the way attributes are parsed so that the "attribute name" after the opening `[` is simply parsed as an identifier. This way we don't have to introduce a new keyword TokenKind for each new attribute that we want to introduce.